### ISSUE TITLE
Ensure django admin panel is accessible

### ISSUE THREAD

**Issue by porta-antiporta** [USKPA-dev/uskpa#32]
The current django admin is not accessible (color contrasts).  @annalee is aware of a plugin/tool that can modify the admin panel to ensure accessibility compliance.

However, that tool does not currently support Django 2.0

- [ ] Modify tool to support Django 2.0
- [ ] Integrate with our application
- [ ] Run accessibility scan to ensure compliance
